app/javascript/packs/complaint-links.js

```diff
@@ -82,18 +82,23 @@ document.querySelectorAll(".js-close-edit-tta").forEach((el) => {
   })
 })
 
+// Unlink form for TTA and Monitoring
+
 document.querySelectorAll(".js-open-unlink-modal").forEach((el) => {
   el.addEventListener('click', event => {
-    const displayId = event.currentTarget.dataset.displayId
-    // add display id to hidden input form
-    const hiddenInput = document.querySelector("#hidden-tta-display_id")
-    hiddenInput.value = displayId
+    // note: id here refers to monitoring's review id or a TTA issue's display_id
+    const id = event.currentTarget.dataset.id
+    const field = event.currentTarget.dataset.inputField
```

Comment:
[suggestion (nonblocking)] what are your thoughts on assignment destructuring?

```suggestion
    const { id, field } = event.currentTarget.dataset
```

Reply:
I like the brevity of it, even if I distrust the order things will get assigned since I am paranoid. I can do it!

Reply:
Oh, also, my suggestion is broken, so don't just accept it :)

`const { id, inputField: field } = event.currentTarget.dataset` should work

Reply:
My distrust was warranted! I will have to look up how the javascript assignment stuff works, because that second solution is a little less clear to me and I definitely wouldn't know how it was working if I hadn't just been working in that code.

Reply:
Fair enough. Here's some docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring